experimental/components/ConversationalCore/assistant.dialog

```diff
@@ -0,0 +1,83 @@
+{
+  "$kind": "Microsoft.AdaptiveDialog",
+  "$designer": {
+    "$designer": {
+      "name": "Assistant",
+      "id": "yljW4K"
+    }
+  },
+  "autoEndDialog": true,
+  "defaultResultProperty": "dialog.result",
+  "triggers": [
+    {
+      "$kind": "Microsoft.OnConversationUpdateActivity",
+      "$designer": {
+        "id": "376720"
+      },
+      "actions": [
+        {
+          "$kind": "Microsoft.BeginDialog",
+          "$designer": {
+            "id": "dSHrlN"
+          },
+          "activityProcessed": true,
+          "dialog": "Greeting"
+        }
+      ]
+    },
+    {
+      "$kind": "Microsoft.OnIntent",
```

Comment:
This is the root bot dialog, assuming that Onboarding exists. I don't think this should be in a package.

Reply:
Removed OnIntent trigger for Onboarding intent.